WalletWasabi.Fluent/Converters/NavBarIconConverter.cs

```diff
@@ -12,20 +12,20 @@ private NavBarIconConverter()
 	{
 	}
 
-	object? IValueConverter.Convert(object value, Type targetType, object parameter, CultureInfo culture)
+	object IValueConverter.Convert(object? value, Type targetType, object? parameter, CultureInfo culture)
 	{
-		if (value is string iconName)
+		if (Application.Current is { } && value is string iconName)
 		{
 			if (Application.Current.Styles.TryGetResource(iconName, out object? resource))
 			{
-				return resource;
+				return resource ?? AvaloniaProperty.UnsetValue;
```

Comment:
I think `[NotNullWhen(true)]` attributes can be added on Avalonia side, and we don't have to deal with null checks.

Reply:
unfortunately not possible because null is a valid value for Brushes, i.e. Transparent and null have different purposes.